packages/eui/src/services/theme/provider.tsx

```diff
@@ -87,8 +87,9 @@ export const EuiThemeProvider = <T extends {} = {}>({
   const parentHighContrastMode = useContext(EuiHighContrastModeContext);
   const parentTheme = useContext(EuiThemeContext);
 
-  const [system, setSystem] = useState(_system || parentSystem);
-  const prevSystemKey = useRef(system.key);
+  // If the user has an OS-wide high contrast theme applied, it will ignore EUI's
+  // colors and light/dark mode. We should respect that use the system setting
```

Comment:
Small typo: `We should respect that use {of} the system setting`

Reply:
Derp, I think I was combining two sentences there and mangled both somehow.
```suggestion
  // colors and light/dark mode. We should respect the user's system setting
```